src/tests/end-to-end/tests/target-page/visualization-boxes.test.ts

```diff
@@ -19,21 +19,18 @@ describe('Target Page visualization boxes', () => {
         await popupPage.gotoAdhocPanel();
     });
 
-    beforeEach(async () => {
```

Comment:
What motivated this change?

Reply:
jest-circus and puppeteer have some inconsistent behavior with `beforeEach` and so I wanted to try and see if removing the code from `beforeEach` in our `e2e` will help or not.